__tests__/enhancement.test.js

```diff
@@ -0,0 +1,23 @@
+
+
+
+//The enhancement level of an item starts at 0.
```

Comment:
I like that you have a lot of comments here. This is a nice project to plan out beforehand.